docs/install/source.md

```diff
@@ -0,0 +1,308 @@
+    ```
+
+3. Switch to the Percona Server for MongoDB branch that you are building
+   and install Python3 modules
+
+    ```{.bash data-prompt="$"}
+    $ cd percona-server-mongodb && git checkout v6.0
+    $ python3 -m pip install --user -r etc/pip/dev-requirements.txt
+    ```
+
+4. Define Percona Server for MongoDB version (6.0.6 for the time of
+   writing this document)
+
+    ```{.bash data-prompt="$"}
+    $ echo '{"version": "6.0.6"}' > version.json
+    ```
+
+#### Install operating system dependencies
+
+=== "Debian/Ubuntu"
+
+    The following command installs the dependencies for Ubuntu 22.04:
+
+    ```{.bash data-prompt="$"}
+    $ sudo apt install -y gcc g++ cmake curl libssl-dev libldap2-dev libkrb5-dev libcurl4-openssl-dev libsasl2-dev liblz4-dev libbz2-dev libsnappy-dev zlib1g-dev libzlcore-dev liblzma-dev e2fslibs-dev
+    ```
+
+=== "RHEL / CentOS"
+
+    The following command installs the dependencies for CentOS 9:
+
+    ```{.bash data-prompt="$"}
+    $ sudo yum -y install gcc gcc-c++ cmake curl openssl-devel openldap-devel krb5-devel libcurl-devel cyrus-sasl-devel bzip2-devel zlib-devel lz4-devel xz-devel e2fsprogs-devel
+    ```
+
+#### Build AWS Software Development Kit for C++ library
+
+1. Clone the AWS Software Development Kit for C++ repository
+
+    ```{.bash data-prompt="$"}
+    $ git clone --recurse-submodules https://github.com/aws/aws-sdk-cpp.git
+    ```
+
+2. Create a directory to store the AWS library 
+
+    ```{.bash data-prompt="$"}
+    $ mkdir -p /tmp/lib/aws
+    ``` 
+
+3.  Declare an environment variable ``AWS_LIBS`` for this directory 
+    ```{.bash data-prompt="$"}
+    $ export AWS_LIBS=/tmp/lib/aws
+    ``` 
+
+4. Percona Server for MongoDB is built with AWS SDK CPP 1.9.379
+   version. Switch to this version 
+
+    ```{.bash data-prompt="$"}
+    $ cd aws-sdk-cpp && git checkout 1.9.379
+    ``` 
+
+5. It is recommended to keep build files outside the SDK directory.
+   Create a build directory and navigate to it 
+
+    ```{.bash data-prompt="$"}
+    $ mkdir build && cd build
+    ``` 
+
+6.  Generate build files using ``cmake`` 
+
+     ```{.bash data-prompt="$"}
+     $ cmake .. -DCMAKE_BUILD_TYPE=Release '-DBUILD_ONLY=s3;transfer' -DBUILD_SHARED_LIBS=OFF -DMINIMIZE_SIZE=ON -DCMAKE_INSTALL_PREFIX="${AWS_LIBS}"
+     ```
+
+7.  Install the SDK 
+
+    ```{.bash data-prompt="$"}
+    $ make install
+    ```
+
+#### Build Percona Server for MongoDB
+
+1. Change directory to ``percona-server-mongodb`` 
+   
+    ```{.bash data-prompt="$"}
+    $ cd percona-server-mongodb
+    ``` 
+
+2. Build Percona Server for MongoDB from ``buildscripts/scons.py``
+     
+    ```{.bash data-prompt="$"}
+    $ buildscripts/scons.py --disable-warnings-as-errors --release --ssl --opt=on -j$(nproc --all) --use-sasl-client --wiredtiger --audit --inmemory --hotbackup CPPPATH="${AWS_LIBS}/include" LIBPATH="${AWS_LIBS}/lib ${AWS_LIBS}/lib64" install-mongod install-mongos install-mongo
```

Comment:
```suggestion
    $ buildscripts/scons.py --disable-warnings-as-errors --release --ssl --opt=on -j$(nproc --all) --use-sasl-client --wiredtiger --audit --inmemory --hotbackup CPPPATH="${AWS_LIBS}/include" LIBPATH="${AWS_LIBS}/lib ${AWS_LIBS}/lib64" install-mongod install-mongos
```

Please remove target mongo, old shell is used just for autotests and customers don't need it